src/Pages/pokemonDetail.jsx

```diff
@@ -0,0 +1,48 @@
+import { useParams } from "react-router-dom";
+import { gql, useQuery } from "@apollo/client";
+import { Link } from "react-router-dom";
+
+const pokemonDetail = gql`
+  query samplePokeAPIquery($PokemonID: Int!) {
```

Comment:
Suggestion: A more meaningful name for the query can be used. Instead of `samplePokeAPIquery` we can call it something like `pokemonDetailQuery` or something else to that effect.

```suggestion
  query pokemonDetailQuery($PokemonID: Int!) {
```